mono/metadata/cominterop.c

```diff
@@ -573,22 +573,16 @@ cominterop_get_interface_checked (MonoComObject* obj, MonoClass* ic, MonoError *
  * Returns: the COM interface requested
  */
 static gpointer
-cominterop_get_interface (MonoComObject *obj, MonoClass *ic, gboolean throw_exception)
+cominterop_get_interface (MonoComObject *obj, MonoClass *ic)
```

Comment:
(bikeshed) rename to `ves_icall_cominterop_get_interface` so it's a little bit easier to tell that it's a JIT icall

Reply:
Sorry, no renames here yet. Given change of interface though..not a bad idea.

I'd like icall-def and jit to be named differently, I think.
Because jit is apparently stuck in old style.
mono_jit_icall_foo ?

I'd suggest every single JIT icall get such a name, even if it is just one line to call some other function, i.e. when there are no pointers.

If people are worried about the cost of the (tail) call, which is legitimate given elf semantics and cross-module calls, use a define, or an inline in a header, yes.